package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "explorer.loopback.io",
+  "version": "1.0.0",
+  "description": "Scripts for the jekyll-based explorer.loopback.io site",
+  "private": true,
+  "scripts": {
+    "start": "http-server -p 4001",
+    "postinstall": "npm run swagger-ui",
+    "swagger-ui": "node ./upgrade-swagger-ui.js"
+  },
+  "engines": {
+    "node": ">=8"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Strongloop/loopback.io.git"
+  },
+  "devDependencies": {
+    "fs-extra": "^7.0.0",
+    "swagger-ui-dist": "^3.19.0"
```

Comment:
The `http-server` dev dependency is missing. Let's make it self-contained instead of requiring a global installation.